docs/guide/dev/integration/vue/README.md

```diff
@@ -329,6 +329,32 @@ Corresponds to the {@linkapi CKEDITOR.editor#event-change `change`} editor event
 <ckeditor @input="onEditorInput"></ckeditor>
 ```
 
+### `namespaceloaded`
+
+Fired once {@linkapi CKEDITOR} namespace is loaded on the page. This event is called only one time for every editor instance loaded on the page and only when using [`editor-url`](#editor-url) directive with valid CKEditor 4 CDN URL. Allows to modify global editor namespace.
```

Comment:
```suggestion
Fired once {@linkapi CKEDITOR} namespace is loaded on the page. This event is called only one time for every editor instance and only when using [`editor-url`](#editor-url) directive with valid CKEditor 4 URL. Allows to modify global editor namespace.
```

Besides removing the repetition, I'd also resign from `CDN` as this should work also for local CKEditor 4 builds.